testdata/dnn/download_models.py

```diff
@@ -168,6 +174,13 @@ def get_confirm_token(response):  # in case of large files
         except Exception as e:
             print('  download {}'.format(e))
 
+    def download_gdown(self):
```

Comment:
So `download_gdrive` cannot handle downloading large file (>100mb)?

Reply:
Yes, it can not

Reply:
What about the following part? https://github.com/opencv/opencv_extra/blob/586e02072645feecc460f377592ee32d87f6a761/testdata/dnn/download_models.py#L165C1-L165C71

AFAIK, GDrive requires authentification for large files. How gdown resolves this problem?